ElasticaAdmin/BlockParameter/ElasticSearchStrategy.php

```diff
@@ -10,6 +10,8 @@
  */
 class ElasticSearchStrategy implements BlockParameterInterface
 {
+    const NAME = 'elastica_search';
```

Comment:
If you want to be coherent with what you've done on ElasticaFront/DisplayBlock/ElasticaSearchStrategy.php, you should alter the getName method too (l38)